### DISCUSSION TITLE
Github Education Benefits

### DISCUSSION THREAD

**Discussion by berkayozitr06** [community/community#132881] [GitHub Education]
### Hi everyone,

Hello. I approved my github education(for students) request. It is written "your benefits will be available soon(within 3 days)". Although 3 days passed on my request, my benefits are not available now. Can you help me ? Thanks in advance.

Best Regards.

**Comment by davevad93** [comment]
Hi @berkayozitr06 ,

It usually takes a few days, but sometimes it may take a little longer; the waiting time may vary depending on different factors, e.g. the volume of applications.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @berkayozitr06 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470
